README.md

```diff
@@ -2,210 +2,133 @@
-- co-register all contrasts to the T2 spinal cord 
-- average all segmentations from each contrast within the same space (the T2)
-- bring back the segmentations to the original image space of each contrast (except for the T2)
+## Table of contents
+* [1. Using contrast agnostic model with SCT](#1-using-contrast-agnostic-model-with-sct)
+* [2. Retraining the contrast-agnostic model ](#2-retraining-the-contrast-agnostic-model)
+* [3. Lifelong learning for monitoring morphometric drift](#3-lifelong-learning-for-monitoring-morphometric-drift)
 
-The output of this script is a new `derivatives/labels_softseg/` folder that contains the soft labels to be used in this contrast-agnostic segmentation project. All the registration were manually QC-ed (see [Quality Control](#quality-control)) and the problematic registrations were listed in [exclude.yml](https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/blob/main/exclude.yml). The processing was run again to generate the soft segmentations. 
+<!-- * [5. Computing morphometric measures (CSA)](#5-computing-morphometric-measures-csa)
+    * [5.1. Using contrast-agnostic model (best)](#51-using-contrast-agnostic-model-best)
+    * [5.2. Using nnUNet model](#52-using-nnunet-model)
+* [6. Analyse CSA and QC reports](#6-analyse-csa-and-qc-reports)
+* [7. Get QC reports for other datasets](#7-get-qc-reports-for-other-datasets)  
+    * [7.1. Running QC on predictions from SCI-T2w dataset](#71-running-qc-on-predictions-from-sci-t2w-dataset)
+    * [7.2. Running QC on predictions from MS-MP2RAGE dataset](#72-running-qc-on-predictions-from-ms-mp2rage-dataset)
+    * [7.3. Running QC on predictions from Radiculopathy-EPI dataset](#73-running-qc-on-predictions-from-radiculopathy-epi-dataset) -->
 
-Specify the path of preprocessed dataset with the flag `-path-data`. 
 
-### 3.1. Launch Preprocessing
+## 1. Using contrast-agnostic model with SCT
 
-This section assumes that SCT is installed. The installation instructions can be found [here](https://spinalcordtoolbox.com/en/latest/user_section/installation.html).
+### Installing dependencies
 
-```
-cd processing_spine_generic
-sct_run_batch -jobs -1 -path-data <PATH_DATA> -path-output <PATH-OUTPUT> -script process_data.sh -script-args exclude.yml
-```
+- [Spinal Cord Toolbox (SCT) v7.0](https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases/tag/7.0) or higher -- follow the installation instructions [here](https://github.com/spinalcordtoolbox/spinalcordtoolbox?tab=readme-ov-file#installation)
+- [conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html) 
+- Python (v3.9)
 
-or use a config file:
+If at the time of installed, SCT v7.0 is not available, download the previous stable version and pull the master branch to get the latest version of the contrast-agnostic model. Once the dependencies are installed, download the latest contrast-agnostic model:
 
+```bash
+sct_deepseg spinalcord -install
```

Comment:
why not simply redirecting to SCT install and usage for that command?

Reply:
good idea! Done here  https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/138/commits/fb7b253fc1643b277f6472d88401f0acc2c7aee7 

(note, at the moment, page does not exist but once the v7.0 is the "stable" version, then the command should work as it currently undergoing the transition from `sct_deepseg -task seg_sc_contrast_agnostic` to `sct_deepseg spinalcord`.